SCClassLibrary/Common/Core/Object.sc

```diff
@@ -501,18 +471,9 @@ Object {
 	}
 
 	// coroutine support
-	yield {
-		_RoutineYield
-		^this.primitiveFailed
-	}
-	alwaysYield {
-		_RoutineAlwaysYield
-		^this.primitiveFailed
-	}
-	yieldAndReset { arg reset = true;
-		_RoutineYieldAndReset
-		^this.primitiveFailed
-	}
+	yield {	^this.sc_abstract_object_yield }
```

Comment:
could you briefly explain why we can't or shouldn't write it here as defined in AbstractObject?
```
yield {	_RoutineYield; ^super.sc_abstract_object_primitive_failed }
```